<ui version="4.0" >
 <class>GroupProperties</class>
 <widget class="QDialog" name="GroupProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>448</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>false</bool>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops" >
   <bool>false</bool>
  </property>
  <property name="windowTitle" >
   <string>Group Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/images.qrc" >
    <normaloff>:/new/prefix1/images/editGroup.png</normaloff>:/new/prefix1/images/editGroup.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Group ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="GIDEdit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="2" >
    <widget class="QToolButton" name="newGroupBtn" >
     <property name="cursor" >
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/images.qrc" >
       <normaloff>:/new/prefix1/images/addgr.jpeg</normaloff>:/new/prefix1/images/addgr.jpeg</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>50</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="3" colspan="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>98</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="1" >
    <widget class="QLineEdit" name="groupNameEdit" >
     <property name="focusPolicy" >
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Group Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="existsLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <layout class="QGridLayout" name="gridLayout" >
     <item rowspan="3" row="0" column="0" >
      <widget class="QTreeView" name="membersList" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor" stdset="0" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="acceptDrops" >
        <bool>true</bool>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="styleSheet" >
        <string notr="true" >selection-background-color: red</string>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="animated" >
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="3" row="0" column="2" >
      <widget class="QTreeView" name="notMembersList" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor" stdset="0" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="acceptDrops" >
        <bool>true</bool>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="styleSheet" >
        <string notr="true" >selection-background-color: red</string>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="animated" >
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="addBtn" >
         <property name="text" >
          <string>&lt;- Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="removeBtn" >
         <property name="text" >
          <string>Remove-></string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" >
      <spacer name="verticalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>98</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>270</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2" colspan="2" >
    <widget class="QPushButton" name="okBtn" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/images.qrc" >
       <normaloff>:/new/prefix1/images/ok.png</normaloff>:/new/prefix1/images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="4" >
    <widget class="QPushButton" name="closeBtn" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/images.qrc" >
       <normaloff>:/new/prefix1/images/cancel.png</normaloff>:/new/prefix1/images/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="GIDLabel" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>16</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>GroupProperties</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>385</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel" >
     <x>188</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
